docs/development.md

```diff
@@ -4,12 +4,18 @@ GLAM is maintained on GitHub at:
 
 https://github.com/mozilla/glam
 
-GLAM uses Docker for local development and deployment. Please make sure to
-install [Docker] and [Docker Compose] on your computer to contribute code or
-documentation changes. Note that GLAM requires a fair bit of disk space-- if
-you're using Docker for Mac, you might want to increase the allocation of disk
-space to the Docker virtual machine to at least 100G (via Preferences ->
-Resources -> Advanced).
+To install a local copy of GLAM, you need:
+
+- [Python](https://www.python.org/) (version 3.8+)
+- [node.js](https://nodejs.org/) (version 12+)
+- [npm](https://docs.npmjs.com/cli/v7/commands/npm) (v7+): run
```

Comment:
This says 7+, but package.json says 8. It looks like we only made the Glean Dictionary require 7, and it works fine:

https://github.com/mozilla/glean-dictionary/blob/main/package.json

Reply:
Yup I think using npm v7+ is good enough. Let me update this.